packages/validations/index.js

```diff
@@ -326,12 +329,16 @@ When('the file {string} is gzipped', function (filename) {
 })
 
 MAFWhen('file {string} is gzip unzipped to file {string}', function (file, fileOut) {
+  if (!this.results){
```

Comment:
The `MAFWhen` function handles this as part of it.

Reply:
Removed added this.results from file